src/main/kotlin/net/onelitefeather/clipboardconnect/services/SyncService.kt

```diff
@@ -66,20 +63,25 @@ class SyncService @Inject constructor(private val config: FileConfiguration, pri
     }
 
     private fun pollUpdates() {
-        val message = messageRQueue.peek()
-        logger.debug(MiniMessage.miniMessage().deserialize("Pull message queue"))
-        if (message != null) {
-            logger.debug(MiniMessage.miniMessage().deserialize("Found message"))
-            val player = Bukkit.getPlayer(message.userId) ?: return
-            logger.debug(MiniMessage.miniMessage().deserialize("Found player"))
-            if (!player.hasPermission("clipboardconnect.service.load")) return
-            logger.debug(MiniMessage.miniMessage().deserialize("Player permission check ok"))
-            if (syncPull(BukkitAdapter.adapt(player))) {
-                logger.debug(MiniMessage.miniMessage().deserialize("Pull was successful"))
-                player.sendMessage(MiniMessage.miniMessage().deserialize("<prefix><green>Clipboard from <gold><server> <green>was successful transfer to this server", Placeholder.unparsed("server", message.fromServer), Placeholder.component("prefix",prefix)))
+        try {
+            val message = messageRQueue.peek()
+            logger.debug(MiniMessage.miniMessage().deserialize("Pull message queue"))
+            if (message != null) {
+                logger.debug(MiniMessage.miniMessage().deserialize("Found message"))
+                val player = Bukkit.getPlayer(message.userId()) ?: return
+                logger.debug(MiniMessage.miniMessage().deserialize("Found player"))
+                if (!player.hasPermission("clipboardconnect.service.load")) return
+                logger.debug(MiniMessage.miniMessage().deserialize("Player permission check ok"))
+                if (syncPull(BukkitAdapter.adapt(player))) {
+                    logger.debug(MiniMessage.miniMessage().deserialize("Pull was successful"))
+                    player.sendMessage(MiniMessage.miniMessage().deserialize("<prefix><green>Clipboard from <gold><server> <green>was successful transfer to this server", Placeholder.unparsed("server", message.fromServer()), Placeholder.component("prefix",prefix)))
```

Comment:
"<prefix><green>Clipboard from <gold><server> <green>was successfully transfered to this server"
(You made some typo there)